apps/web_fe/views.py

```diff
@@ -565,18 +576,52 @@ def velo_save_file(request):
     if request.method == 'POST':
         try:
             incomming_file = json.loads(request.body)
-            remote_path = incomming_file['remote_path']
+            # remote_path = incomming_file['remote_path']
             filename = incomming_file['filename']
             text = incomming_file['text']
-            site_user = 'acmetest'
-            velo_user = 'acmetest'
-            velo_pass = 'acmetest'
+
+            cred = Credential.objects.get(
+                site_user_name=request.user, service="velo")
+
+            process = Popen(
+                ['python', './apps/velo/save_file.py', text, filename, cred.site_user_name, cred.service_user_name, cred.password], stdout=PIPE)
+            (out, err) = process.communicate()
+            exit_code = process.wait()
+            if exit_code >= 0 and 'File saved' in out:
+                return HttpResponse(status=200)
+            else:
+                print out, err
+                return HttpResponse(status=500)
+        except Exception as e:
+            import traceback
```

Comment:
If you really want all of this debugging logging stuff, put it in a "debug_exception" function and save yourself some copy-pasta.